### ISSUE TITLE
Catapult and perf need a better workflow for finding references to closed bugs in the code

### ISSUE THREAD

**Issue by zeptonaut** [catapult-project/catapult#3615]
@randalnephew sent me a code review today that moves some old story disables on loading.py over to his awesome new method of story disabling.

The disables that got moved over each had a bug associated with them, and literally every single one of the 10 or so disables was linked to a bug that was already closed. I've seen similar stuff happen in the past, too. 

It'd be great to have a tool that helps us find I think that eventually, the solution to this is that the new tricium project in Chromium is going to be able to add annotations to code search that identifies stale bugs (see https://goto.google.com/kjfgvjhywjfmn). In the short-term, though, it'd be great to have a short-term workaround. It seems like something could be hacked together with Python using grep and the monorail API without too much time investment. 

@nedn do you know of any easy way to do this already?

**Comment by nedn** [comment]
@dpranke this seems like a general problem for Chrome tests.

**Comment by dpranke** [comment]
It's sort of a general problem, as in every test suite that allows you disable tests and associate them with bugs may suffer from this problem, but given that we do disabling so many different ways, it doesn't seem like there's a single implementation that will solve the problem.

For example, telemetry/catapult tends to use unittest annotations and other mechanism to disable things, which I don't think we tend to do anywhere else.

**Comment by nedn** [comment]
Can flakiness dashboard support DISABLED test field with a crbug number? If so, we can add crbug number for corresponding disabled test in the generated json result.

**Comment by dpranke** [comment]
Yes, the json results format supports bug numbers and disabled (i.e., skip) fields, but you'd have to make sure you had some way to distinguish intentionally disabled due to a bug from intentionally disable due to a runtime mismatch (i.e., android-only test, or some such).

**Comment by nedn** [comment]
@randalnephew fyi

Seems like one the data is there, we can make flakiness dashboard shows some warning if we disable a stale bug.
